packages/s3-cache/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { S3 } from '@aws-sdk/client-s3'
```

Comment:
Would it makes sense to use our wrapper? https://github.com/DBB-Software/platform/blob/main/packages/common/src/aws/s3.ts
Or at least logger aws-xray-sdk-core

Reply:
this has to be a published package https://github.com/DBB-Software/platform/blob/main/packages/common/src/aws/s3.ts, + we really need 2 methods from s3 library at this moment